client/src/components/EditProfile/EditPhoto/useStyles.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  authWrapper: {
+    display: 'flex',
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    minHeight: '100%',
+    paddingBottom: 40,
+  },
+  title: {
+    padding: 40,
+  },
+  reminder: {
+    padding: 20,
+    width: '16vw',
+  },
+  centerRow: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  avatar: {
+    margin: 10,
+    width: 140,
+    height: 140,
+    alignSelf: 'center',
+  },
+  uploadButton: {
+    textTransform: 'none',
+    fontSize: 12,
+    fontWeight: 700,
```

Comment:
Similarly, relying on theme for font properties is usually a better idea. For example, the Button component comes with a [`size` prop](https://v4.mui.com/customization/spacing/) which has 3 variants: small, medium and large. You can [customize](https://v4.mui.com/customization/globals/#css) each to suit your needs and use them throughout your app, e.g:
```
{
...
   overrides: {
      MuiButton: {
        sizeSmall: {
          fontSize: "12px",
        },
        sizeLarge: {
         fontSize: "32px",
        },
      }
    }
```

Following this practice changes the decision you face from "What should be the font size of the button here?" to "What should be the size of this button?" and therefore makes it a much more constrained decision. Remaining consistent is much easier when you have 3 options to chose from 😄. You can still apply styles and font sizes through css when there needs to be an exception 👍🏼

Reply:
I agree. I will use the theme instead.